### ISSUE TITLE
which direction is top layer?

### ISSUE THREAD

**Issue by ludlows** [brunoklein99/srcnn#3]
according to 'multipliers=[1, 1, 1, 1, 0.1, 0.1]'

do you mean the top layer is corresponding to the first multiplier?

In keras source code, where can I find the proof to support this idea?

Thanks in advance.

**Comment by brunoklein99** [comment]
https://github.com/brunoklein99/srcnn/blob/master/LRMultiplierSGD.py#L47

This is not Keras's, it's a custom implementation, as per above.

1st multiplier is first layer weights, 2nd is first layer bias, 3rd is second layer weights and so on...

**Comment by ludlows** [comment]
Yes. I see. 
Could you give me a hint about how to find the layer order in a NN architecture?
Because I want to set the learning rate of top layer as a bigger one, and decrease learning rate layer by layer.
Thanks in advance.

**Comment by brunoklein99** [comment]
If I'm not mistaken, `grads` is a list, which is sorted in "layer order".

https://github.com/brunoklein99/srcnn/blob/master/LRMultiplierSGD.py#L35

You can debug and verify.

**Comment by ludlows** [comment]
ok.thanks.
